react/SlideToggle/SlideToggle.less

```diff
@@ -0,0 +1,82 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  display: inline-flex;
+  line-height: @grid-row-height * 3;
+  height: @grid-row-height * 3;
+  font-size: 14px;
+}
+
+.switch {
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+}
+
+.input {
+  position: absolute;
+  left: 10px;
+  top: 5px;
+  margin: 0;
+
+  &:focus + .slider {
+    box-shadow: 0px 0px 10px 0px @sk-focus;
+  }
+
+  &:active + .slider .slideButton {
+    transform: scale(1.15, 0.85);
```

Comment:
Why are performing a scale here? This causes the switch to shrink when you click on it, that includes the icon which I feel looks a bit weird. What effect are we going for exactly?
From
![image](https://cloud.githubusercontent.com/assets/8802980/26087394/e87c33ac-3a33-11e7-8ac6-b54d8f8dc724.png)

To
![image](https://cloud.githubusercontent.com/assets/8802980/26087385/d925bbbc-3a33-11e7-9a2c-49689e586e72.png)